container/kvm/kvm.go

```diff
@@ -26,20 +29,29 @@ import (
 var (
 	logger = loggo.GetLogger("juju.container.kvm")
 
+	// KvmObjectFactory is the container factory for kvm containers.
```

Comment:
This is pre-existing code. Horatio asked for comments here. So I just added comments. I didn't actually get up into the container factory and container interface code that is used by the worker/provisioner/kvm-broker code. I mostly made changes that removed the need for uvtool but left the API untouched.